test/controllers/patients_controller_test.rb

```diff
@@ -123,4 +123,30 @@ class PatientsControllerTest < ActionController::TestCase
       assert_redirected_to root_path
     end
   end
+
+  #confirm get :data_entry returns a success code
+  describe `data_entry method` do
+    before do
+      @new_patient = attributes_for :patient, name: 'Test Patient'
+      @new_patient[:pregnancy] = attributes_for :pregnancy
+    end
+
+    it 'should respond success on completion' do
+      assert_response do get :data_entry, :success
```

Comment:
Ah, I see what's up. You're super close -- let's take a look at the documentation for `assert_response` here -- http://guides.rubyonrails.org/testing.html#helpers-available-for-integration-tests -- check out section 5.2.1. The nut of it is here: 

```
  get "/articles/new"
  assert_response :success
```

the format here is slightly different but pretty close! Basically, you want to switch this into two statements; one that does a get, and then one that asserts it has a successful response.